### ISSUE TITLE
Visual studio code does not accept phpcbf

### ISSUE THREAD

**Issue by fbsanchez** [soderlind/vscode-phpcbf#43]
Hi folks,

![image](https://user-images.githubusercontent.com/12999095/136775116-87cba696-8790-4409-9918-70ed61b48005.png)

Received this message after upgrade vscode to 1.61 (fedora) confirmed in multiple devices/projects, so seems not to be configuration issues.

If there's any new configuration flag we could miss or any advice or help, we'll be very grateful!

Thanks

**Comment by pattyok** [comment]
I had this same issue and found myself here... for me restarting Vscode again fixed it.

**Comment by fbsanchez** [comment]
> I had this same issue and found myself here... for me restarting Vscode again fixed it.

Yes, after each update seems to fail, but after a reload works again
